SPECS/dotnet-runtime-3.1/dotnet-runtime-3.1.spec

```diff
@@ -1,96 +1,81 @@
-ExclusiveArch:  x86_64
+URL:            https://github.com/dotnet/core
 Source0:        https://download.visualstudio.microsoft.com/download/pr/d00eaeea-6d7b-4e73-9d96-c0234ed3b665/0d25d9d1aeaebdeef01d15370d5cd22b/dotnet-runtime-3.1.5-linux-x64.tar.gz
-
+Requires:       glibc
 Requires:       icu
-Requires:       libunwind
 Requires:       krb5
+Requires:       libgcc
+Requires:       libstdc++
+Requires:       libunwind
 Requires:       lttng-ust
 Requires:       openssl
 Requires:       zlib
-Requires:       glibc
-Requires:       libgcc
-Requires:       libstdc++
+ExclusiveArch:  x86_64
 
 %description
 .NET Core is a development platform that you can use to build command-line
 applications, microservices and modern websites.
 
 %prep
-%setup -qc dotnet-runtime-%{version}
+%setup -qc -T -a 0 dotnet-runtime-%{version}
 
 %build
 
 %install
 mkdir -p %{buildroot}%{_libdir}/dotnet
 mkdir -p %{buildroot}%{_docdir}/dotnet-runtime-%{version}
-cp LICENSE.txt ThirdPartyNotices.txt %{buildroot}%{_docdir}/dotnet-runtime-%{version}
-rm LICENSE.txt ThirdPartyNotices.txt
+
 cp -r * %{buildroot}%{_libdir}/dotnet
+rm %{buildroot}%{_libdir}/dotnet/LICENSE.txt %{buildroot}%{_libdir}/dotnet/ThirdPartyNotices.txt
 mkdir -p %{buildroot}%{_bindir}
 ln -sf %{_libdir}/dotnet/dotnet %{buildroot}%{_bindir}/dotnet
 
-# Pre-install
-%pre
-
-    # First argument is 1 => New Installation
-    # First argument is 2 => Upgrade
-
-# Post-install
-%post
-
-    # First argument is 1 => New Installation
-    # First argument is 2 => Upgrade
-
-    /sbin/ldconfig
-
-# Pre-uninstall
-%preun
-
-    # First argument is 0 => Uninstall
-    # First argument is 1 => Upgrade
-
-# Post-uninstall
-%postun
-
-    /sbin/ldconfig
-
-    # First argument is 0 => Uninstall
-    # First argument is 1 => Upgrade
+%post -p /sbin/ldconfig
+%postun -p /sbin/ldconfig
 
 %files
-%license %{_docdir}/dotnet-runtime-%{version}/LICENSE.txt
+%license LICENSE.txt ThirdPartyNotices.txt
 %defattr(-,root,root,0755)
 %exclude %{_libdir}/debug
 %{_docdir}/*
 %{_bindir}/dotnet
 %{_libdir}/*
 
 %changelog
+* Thu Nov 12 2020 Henry Beberman <henry.beberman@microsoft.com> - 3.1.5-2
+- Fix scriptlets and move licenses to the correct folder
+
 *   Fri Jun 19 2020 Andrew Phelps <anphel@microsoft.com> 3.1.5-1
 -   Update version to 3.1.5. Fix runtime requirements.
+
 *   Sat May 09 2020 Nick Samson <nisamson@microsoft.com> 3.1.3-4
 -   Added %%license line automatically
+
 *   Tue May 05 2020 Emre Girgin <mrgirgin@microsoft.com> 3.1.3-3
 -   Renaming dotnet-runtime to dotnet-runtime-3.1
+
 *   Mon May 04 2020 Emre Girgin <mrgirgin@microsoft.com> 3.1.3-2
 -   Replace BuildArch with ExclusiveArch
+
 *   Mon Mar 30 2020 Andrew Phelps <anphel@microsoft.com> 3.1.3-1
 -   Update to dotnet 3.1.3. License verified.
+
 *   Tue Sep 03 2019 Mateusz Malisz <mamalisz@microsoft.com> 2.2.0-2
--   Initial CBL-Mariner import from Photon (license: Apache2).
+-   Initial import from Photon (license: dual Apache2/GPL2).
```

Comment:
The previous version was correct. If it's linter's doing, let me know (or go directly to Thomas).

Reply:
This ones on me. Copied the spec over from a previous proof of concept before this formatting was changed.